<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/IN.png</normaloff>:/res/IN.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 242, 60);
background-color: rgb(152, 80, 71);
background-color: rgb(44, 46, 67);
background-color: rgb(201,201, 201);
border-radius:30px</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widgetALL" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(44, 46, 67);
border-radius:30px</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QHBoxLayout" name="hLayoutALL" stretch="1,10,4">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="vLleft" stretch="1,0,8">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>57</width>
              <height>13</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>9</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <widget class="QLabel" name="labelLogo">
               <property name="minimumSize">
                <size>
                 <width>70</width>
                 <height>70</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius: 10px;</string>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="midLineWidth">
                <number>0</number>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="res.qrc">:/res/IN.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widgetLeftButtons" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout" stretch="8,6">
                <item>
                 <layout class="QVBoxLayout" name="VlayoutLeftButtons" stretch="5,1,5,1,5,1,5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="btnHome">
                    <property name="maximumSize">
                     <size>
                      <width>45</width>
                      <height>45</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
border-image: url(:/res/Home.png);
}
QPushButton:checked{
border-image: url(:/res/HomeOpen.png);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>68</width>
                      <height>68</height>
                     </size>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnDev">
                    <property name="maximumSize">
                     <size>
                      <width>45</width>
                      <height>45</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
border-image: url(:/res/light.png);
}
QPushButton:checked{
border-image: url(:/res/lingOpen.png);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnCam">
                    <property name="maximumSize">
                     <size>
                      <width>45</width>
                      <height>45</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
border-image: url(:/res/savety.png);
}
QPushButton:checked{
border-image: url(:/res/savetyOpen.png);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="btnTheme">
                    <property name="maximumSize">
                     <size>
                      <width>45</width>
                      <height>45</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
border-image: url(:/res/notice.png);
}
QPushButton:checked{
border-image: url(:/res/noticeOpen.png);
}</string>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widgetCenter" native="true">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(23, 27, 34);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="hLayoutCenter" stretch="19,18">
             <property name="spacing">
              <number>15</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="vLayoutPicAnddev" stretch="3,5">
               <item>
                <widget class="ImageSlider" name="widgetShowCen01" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>400</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border-image: url(:/res/bg.jpg);</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="btnVideo">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="minimumSize">
                     <size>
                      <width>21</width>
                      <height>0</height>
                     </size>
                    </property>
                    <property name="layoutDirection">
                     <enum>Qt::LeftToRight</enum>
                    </property>
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>100</width>
                      <height>46</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="ContrlUtils" name="widgetCen02" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>400</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="vLayoutWeaAndMusic" stretch="0,0">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="WeatherWdiget" name="widgetWea" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>700</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="MusicUtils" name="widgetMusic" native="true">
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>200</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="VLR" stretch="0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="LeftCircle" name="widgetR03" native="true">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>400</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="cirtainUtils" name="widgetCur" native="true">
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>600</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>9</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageSlider</class>
   <extends>QWidget</extends>
   <header>imageslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeatherWdiget</class>
   <extends>QWidget</extends>
   <header>weatherwdiget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicUtils</class>
   <extends>QWidget</extends>
   <header location="global">musicutils.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LeftCircle</class>
   <extends>QWidget</extends>
   <header location="global">leftcircle.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ContrlUtils</class>
   <extends>QWidget</extends>
   <header location="global">contrlutils.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>cirtainUtils</class>
   <extends>QWidget</extends>
   <header location="global">cirtainutils.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
